packages/core/admin/admin/src/content-manager/components/BlocksEditor/index.js

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+
+import { Box, Flex, Typography, InputWrapper, Divider } from '@strapi/design-system';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { createEditor } from 'slate';
+import { withHistory } from 'slate-history';
+import { Slate, withReact, ReactEditor } from 'slate-react';
+import styled from 'styled-components';
+
+import BlocksInput from './BlocksInput';
+import { BlocksToolbar } from './Toolbar';
+
+const TypographyAsterisk = styled(Typography)`
+  line-height: 0;
+`;
+
+const EditorDivider = styled(Divider)`
+  background: ${({ theme }) => theme.colors.neutral200};
+`;
+
+const Wrapper = styled(Box)`
+  width: 100%;
+  max-height: 512px;
+  overflow: auto;
+  padding: ${({ theme }) => `${theme.spaces[3]} ${theme.spaces[4]}`};
+  font-size: ${({ theme }) => theme.fontSizes[2]};
+  background-color: ${({ theme }) => theme.colors.neutral0};
+  color: ${({ theme }) => theme.colors.neutral800};
+  line-height: ${({ theme }) => theme.lineHeights[6]};
+  border-radius: ${({ theme }) => theme.borderRadius};
```

Comment:
Loads of these can be passed as props – how come we're not doing this?

Reply:
Ok, I am gonna create a new PR to handle your comments, thanks